src/plugins/Polls/types.ts

```diff
@@ -5,8 +5,16 @@ export interface PollMetaData {
     end_time: number
     options: string[]
     results: number[]
-    sender?: string
-    id?: string
+    sender_name?: string
+    sender_id?: string
+    voting_history?: VotingDetail[]
```

Comment:
This seems strange. How could you upgrade the data that has been serialized into the post?